<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="SideBar" name="widget" native="true"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="labelPredio">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Pr√©dio:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="choosePredio">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>152</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>10000</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelSala">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Sala:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="chooseSala">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>152</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>10000</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTableWidget" name="tableSalas"/>
        </item>
        <item>
         <widget class="QTableWidget" name="tableNAlocados"/>
        </item>
        <item>
         <widget class="QTableWidget" name="tableAlocar"/>
        </item>
        <item>
         <widget class="QTableWidget" name="tablePedidos"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="initButton">
        <property name="text">
         <string>Iniciar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionRemove"/>
   <addaction name="actionImport"/>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icon3</normaloff>:/icons/icon3</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Adds one element</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icon4</normaloff>:/icons/icon4</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove Selected</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/iconImport.jpeg</normaloff>:/icons/icons/iconImport.jpeg</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Import from JSON</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SideBar</class>
   <extends>QWidget</extends>
   <header>sidebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
